### ISSUE TITLE
Not reading USAGE_LOGGERS_URL as system property

### ISSUE THREAD

**Issue by robfromboulder** [resurfaceio/logger-java#58]
This works:

```
USAGE_LOGGERS_URL="..." java -jar target/main-jar-with-dependencies.jar
```

But this doesn't work:

```
PORT=9000 java -DUSAGE_LOGGERS_URL="..." -jar target/main-jar-with-dependencies.jar
```

Both of these forms should work according to the Java API docs (https://github.com/resurfaceio/logger-java/blob/master/API.md)

**Comment by robfromboulder** [comment]
https://www.baeldung.com/java-system-get-property-vs-system-getenv

**Comment by robfromboulder** [comment]
Fixed in 1.9.6
